vg/draw/canvas.go

```diff
@@ -26,14 +26,55 @@ type Canvas struct {
 }
 
 // TextStyle describes what text will look like.
+// Alignment and rotation are interrelated: when
```

Comment:
I don't think having this text here really improves the readability of this. The use of examples in type doc comments is not something that we see in most packages.